### DISCUSSION TITLE
Trouble renewing my domain

### DISCUSSION THREAD

**Discussion by savicaleksa** [vercel/community#5777] [Help]
My domain savicaleksa.dev is set to expire today. I've received an email around a week ago saying that there was a problem with the payment. At that time I didn't have the funds for my domain on my card, because I thought I would be charged today and not a week before the expiration date. After that, I deposited the amount I needed to the card and kept receiving emails from Vercel about payment issues.
I've contacted my bank and they've said that there was only one attempt for payment from Vercel on Feb 1st, even though I kept receiving the emails.
I've also added another card today, but nothing has changed.
I really don't want to lose my domain, so please, can someone tell me what should I do and is there a way to manually pay for it before it expires?

Thanks in advance.

**Comment by amyegan** [comment]
Hey @savicaleksa. If you [submit a case](https://vercel.com/help#issues) to the Customer Success Team, they'll be able to help you got this billing issue fixed 🙇 

[vercel.com/help](https://vercel.com/help)

**Reply by savicaleksa** [reply]
Thanks for the reply, but there's no point now since I've got my ownership revoked this morning. I've already bought a similar one so everything's okay :D

**Reply by amyegan** [reply]
I'm glad you were able to get a similar domain as a workaround. There is a [grace period](https://www.icann.org/resources/pages/errp-2013-02-28-en) during which you can still get a recently expired domain back. If you still want it, just submit a case to get help with late renewal.
